doc/pages/Manual.md

```diff
@@ -457,20 +631,21 @@ The following are treated slightly differently:
   requires files to specify an absolute destination, and the user will be
   prompted before the installation is done.
 
-### opam
+#### opam
 
-The main file specifying a package's metadata. The `opam lint` command is
-recommended to check the validity and quality of your `opam` files.
+The main file specifying a package's metadata. Usage of the `opam lint` command
+is recommended to check the validity and quality of your `opam` files.
 
-`opam` files allow the following fields:
+`opam` files allow the following fields and sections:
 
 - <a id="opamfield-opam-version">`opam-version: <string>`</a> (mandatory):
-  the file format version, should be `1.2` as of writing.
+  the file format version, should be `2.0` as of writing.
 
-- <a id="opamfield-name">`name: <string>`</a>,
-  <a id="opamfield-version">`version: <string>`</a>:
+- <a id="opamfield-name">`name: <pkgname>`</a>,
+  <a id="opamfield-version">`version: <version>`</a>:
   the name and version of the package. Both fields are optional when they can be
-  inferred from the directory name (e.g. when the file sits in the repository).
+  inferred from the directory name (e.g. when the file sits in the repository),
+  but should always be set for package definitions within source trees.
```

Comment:
Hmm, maybe that's not good advice. It allows `opam pin add project-dir/` to pick the version up though...